en/flight_modes_fw/mission.md

```diff
@@ -0,0 +1,378 @@
+# Mission Mode (Fixed-wing)
+
+[<img src="../../assets/site/position_fixed.svg" title="Global position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+_Mission mode_ causes the vehicle to execute a predefined autonomous [mission](../flying/missions.md) (flight plan) that has been uploaded to the flight controller.
+The mission is typically created and uploaded with a Ground Control Station (GCS) application like [QGroundControl](https://docs.qgroundcontrol.com/master/en/) (QGC).
+
+:::note
+
+- This mode requires a global 3d position estimate (from GPS or inferred from a [local position](../ros/external_position_estimation.md#enabling-auto-modes-with-a-local-position)).
+- The vehicle must be armed before this mode can be engaged.
+- This mode is automatic - no user intervention is _required_ to control the vehicle.
+- RC control switches can be used to change flight modes on any vehicle.
+- RC stick movement in a multicopter will [by default](#COM_RC_OVERRIDE) change the vehicle to [Position mode](../flight_modes_mc/position.md) unless handling a critical battery failsafe.
+
+:::
+
+## Description
+
+Missions are usually created in a ground control station (e.g. [QGroundControl](https://docs.qgroundcontrol.com/master/en/PlanView/PlanView.html)) and uploaded prior to launch.
+They may also be created by a developer API, and/or uploaded in flight.
+
+[Mission commands](#mission-commands) are handled in a way that is appropriate for each fixed-wing flight characteristics (for example loiter is implemented as flying in a circle).
+
+:::note
+Missions are uploaded onto a SD card that needs to be inserted **before** booting up the autopilot.
+:::
+
+At high level all vehicle types behave in the same way when MISSION mode is engaged:
+
+1. If no mission is stored, or if PX4 has finished executing all mission commands, or if the [mission is not feasible](#mission-feasibility-checks):
+
+   - If flying the vehicle will loiter.
+   - If landed the vehicle will "wait".
+
+1. If a mission is stored and PX4 is flying it will execute the [mission/flight plan](../flying/missions.md) from the current step.
+   - A takeoff mission item will be treated as a normal waypoint.
+1. If a mission is stored and the vehicle is landed it will not automatically take off (the autopilot will detect the lack of movement and set the throttle to zero).
+   If the currently active waypoint is a Takeoff, the system will automatically takeoff (see [FW Takeoff/Landing in Mission](#mission-takeoff)).
```

Comment:
@sfuhrer These seem mutually exclusive. I think what is true here is that an FW that requires catapult/hand launch will not take off automatically but one that is configured for runway takeoff would?

Reply:
A catapult one will not because it will not even detect takeoff unless the WP type is set to Takeoff, correct.
A runway plane only executes the takeoff logic if the current WP is a Takeoff. If not it may still takeoff somehow, but it's definitely not something we should document beside "make sure that the active WP is a Takeoff one when starting the mission from the vehicle still on the ground".

Reply:
OK, rewriting like this to just mention the requirement for Takeoff mode being active.

```suggestion
1. If a mission is stored and the vehicle is landed it will only takeoff if the active waypoint is a `Takeoff`.
   If configured for catapult launch, the vehicle must also be launched (see [FW Takeoff/Landing in Mission](#mission-takeoff)).
```